src/wallet/asyncrpcoperation_mergetoaddress.cpp

```diff
@@ -253,8 +253,13 @@ bool AsyncRPCOperation_mergetoaddress::main_impl()
     }
     tx_ = CTransaction(rawTx);
 
-    LogPrint(isPureTaddrOnlyTx ? "zrpc" : "zrpcunsafe", "%s: spending %s to send %s with fee %s\n",
-             getId(), FormatMoney(targetAmount), FormatMoney(sendAmount), FormatMoney(minersFee));
+    if (isPureTaddrOnlyTx) {
```

Comment:
Unfortunately, now the category must be a constant string (which is enforced in the `LogPrint()` macro). Luckily there are only two places where it's not.